pkgs/development/python-modules/Flask-RESTful/default.nix

```diff
@@ -0,0 +1,23 @@
+{
+stdenv, pkgs, python, wrapPython, buildPythonPackage,
```

Comment:
just pass `lib` instead of `stdenv`. Furthermore, never pass the whole package set `pkgs`, instead pass individual applications/libraries, like in this case `fetchurl`. `wrapPython` is not used so you don't need to pass it, and neither should you pass `python`.